src/pudl/transform/epacems.py

```diff
@@ -49,12 +49,10 @@ def fix_up_dates(df, plant_utc_offset):
     # Some of the timezones in the plants_entity_eia table may be missing,
     # but none of the CEMS plants should be.
     if not df["utc_offset"].notna().all():
-        missing_plants = df.loc[df["utc_offset"].isna(),
-                                "plant_id_eia"].unique()
-        raise ValueError(
-            f"utc_offset should never be missing for CEMS plants, but was "
-            f"missing for these: {str(list(missing_plants))}"
-        )
+        # In order to process 2020 EPACEMS data with 2020 860m data, we will just
+        # replace any missing utc_offsets with a 0 offset
+        df["utc_offset"].fillna(value=pd.Timedelta(seconds=0), inplace=True)
```

Comment:
Kind of a hack right now, unsure if there would be a better solution to this.

Reply:
Do we know what is the reason for the missing `utc_offsets` here? 

I'm a little reluctant to "silently fix missing data", especially if this is specifically singled out by the code as something that should not happen.

Additionally, using +0 utc_offset seems like a bad default as this would effectively place those plants to Britain timezone which is most likely not appropriate.

Reply:
oh for sure, I would definitely want to figure out why we're actually not seeing offsets here and find a correct solution, I can get some plant id's that trigger this to take a look at

Reply:
I can't recall if we already talked about this in the Slack, but there's already a built-in hack to deal with plants which appear in CEMS but do not appear in the EIA 860/923 data. Mostly it's used to deal with plants from the beforetimes -- since CEMS goes back further than the EIA data we currently have. You can manually specify the plant name, latitude and longitude of a given `plant_id_eia` in `src/pudl/package_data/epa/cems/plant_info_for_additional_cems_plants.csv` (see [this file](https://github.com/catalyst-cooperative/pudl/blob/main/src/pudl/package_data/epa/cems/plant_info_for_additional_cems_plants.csv)) and then its timezone will be set appropriately.

Reply:
oh right. I completely forgot about this. I took a look and added the missing plant info to that file and it works fine now. What I noticed is that if I'm trying to load up only CEMS 2020 data, this plant is missing, but if I add EIA860 data for 2018, it has that plant info and works fine.

Reply:
Ah yes, all bets are off if you're processing a non-standard collection of years / tables. We test that the most recently integrated year works by itself (currently 2019) with all the tables included, and we test that all of the integrated data for all of the tables and datasets work together, but if you have a small collection of years or only some of the datasets, it is very likely that the ETL will fail because there are these relationships between the datasets, and a bunch of checks that values which are reported in multiple places that *should* be the same meet some minimum threshold of self-consistency, with the most commonly reported value being the one that ends up in the well normalized database tables.

In this case, you might want to go ahead and load all of the EIA data, plus whatever subset of the CEMS data you need.

I'm not sure what happens right now if a plant's lat/lon is specified both in the "extra info" file, and in the original data.  Re-specifying that information when it really does exist in the EIA data is probably not ideal.

Reply:
Yeah good point. Our pipeline will include multiple years anyway, so if we're loading it all in then it should be fine either way.